box_it_script/boxit.js

```diff
@@ -0,0 +1,72 @@
+#!/Users/yeosujin/.nvm/versions/node/v14.20.0/bin/node
```

Comment:
JFYI. this only works if someone were to run `./boxit.js ...` off this computer.